.github/scripts/test-project.sh

```diff
@@ -28,6 +28,23 @@ echo ""
 echo "-----------------------------"
 echo "running $dir/$project"
 
+
+# Find schema, if it contains `env("DATABASE_URL")`, db push that schema to database
+if [[ "$project" = "planetscale" ]]
+then
+  true
+  # have to skip planetscale until https://github.com/planetscale/cli/issues/322 is implemented
+else
+  schema_path=$(find $dir/$project -name "schema.prisma" ! -path "*/node_modules/*" | head -n 1)
+  if grep -q "env(\"DATABASE_URL\")" "$schema_path"; then
+    echo ""
+    echo "found 'schema.prisma' with 'env(\"DATABASE_URL\")': $schema_path"
+    echo "npx prisma db push --accept-data-loss --skip-generate --schema=$schema_path"
+    npx prisma db push --accept-data-loss --skip-generate --schema=$schema_path
+    echo ""
+  fi 
```

Comment:
This is the logic that ensures if `DATABASE_URL` is used in the schema, we `db push` that database so it exists.